src/main/java/Nick_Maven/Framework/service/ExpectedParamsCreatorService.java

```diff
@@ -0,0 +1,20 @@
+package Nick_Maven.Framework.service;
+
+import Nick_Maven.Framework.model.ParamsCloudGoogleCalculator;
+
+public class ExpectedParamsCreatorService {
```

Comment:
This classic 'Factory'. Rename this class to 'ParamsCloudGoogleCalculatorFactory' and move them to the model folder.